nix/emanote.nix

```diff
@@ -54,6 +54,16 @@ in
                         description = ''Allow broken links in the static site'';
                         default = false;
                       };
+                      baseUrl = mkOption {
+                        type = types.str;
+                        description = ''Base URL for relative links'';
+                        default = "/";
+                      };
+                      urlStrategy = mkOption {
+                        type = types.str;
+                        description = ''Whether links ends with .html (direct) or not (pretty)'';
+                        default = "direct";
+                      };
```

Comment:
Since there are only two values possible, it would be better to make this a `types.bool`; what do you think?